WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationSettingsViewController.swift

```diff
@@ -210,13 +210,23 @@ class NotificationSettingsViewController: UIViewController {
 
     // MARK: - Error Handling
 
-    fileprivate func handleLoadError() {
+    fileprivate func handleLoadError(error: NSError?) {
         let title = NSLocalizedString("Oops!", comment: "An informal exclaimation meaning `something went wrong`.")
-        let message = NSLocalizedString("There has been a problem while loading your Notification Settings",
-                                            comment: "Displayed after Notification Settings failed to load")
         let cancelText = NSLocalizedString("Cancel", comment: "Cancel. Action.")
         let retryText = NSLocalizedString("Try Again", comment: "Try Again. Action")
 
+        var message = NSLocalizedString(
+            "There has been a problem while loading your Notification Settings",
+            comment: "Displayed after Notification Settings failed to load"
+        )
+
+        if let error {
+            message = NSLocalizedString(
```

Comment:
This localized string should contain only the format:

```swift
let format = NSLocalizedString("notificationSettings.errorMessageFormat", value: "There has been a problem while loading your Notification Settings: %@", comment: "")
let message = String(format:  format, error.localizedDescription)
```

But for simplicity, and to avoid introducing new keys, I'd suggest simply showing `error.localizedDescription` as a message. The title could say "Failed to load settings".

Reply:
Addressed in 0c4c7cef4ee7698db4747e85daf2e1d4bbed4b13

Reply:
And 3e10735a75c3acecdaff04850767781333837143 🤦